main.go

```diff
@@ -1,18 +1,18 @@
 package main
```

Comment:
Bug: import statement for "encoding/json" was removed. Was this intentional or an oversight?